src/skeptic/lib.rs

```diff
@@ -652,6 +652,14 @@ pub mod rt {
         }
     }
 
+    fn get_edition<P: AsRef<Path>>(path: P) -> Result<String> {
+        let path = path.as_ref().join("Cargo.toml");
+        let manifest = cargo_metadata::metadata(Some(&path))?;
+        // FIXME: We have no guarantee that the first entry is the one being tested.
+        let package = &manifest.packages[0];
```

Comment:
How about we iterate over all packages and return the highest `edition` if found. 
Making the return type `Option<String>`
Not being able to read manifest is always a fatal error in context of this script so I'd not care about returning `Result<...>` here

Reply:
Well, there's no need for an `Option<String>`. `cargo_manifest` will always give an edition, whether it finds that key or not. If the key is not in Cargo.toml, then it must be set equal to 2015.
Otherwise sure, I'll implement that.